linera-rpc/proto/rpc.proto

```diff
@@ -51,6 +51,9 @@ service ValidatorNode {
 
   // Request the node's version info.
   rpc GetVersionInfo(google.protobuf.Empty) returns (VersionInfo);
+  
+  // Downloads a blob.
+  rpc DownloadBlob(BlobId) returns (HashedBlob);
```

Comment:
I'd expect a `Blob` to be returned for two reasons:
* The downloader knows the blob ID
* The downloader will probably have to check the hash.